components/patient-data/api/src/main/java/org/phenotips/data/Gene.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses/
+ */
+package org.phenotips.data;
+
+import org.xwiki.stability.Unstable;
+
+import org.json.JSONObject;
+
+/**
+ * Information about a specific gene recorded for a {@link Patient patient}.
+ *
+ * @version $Id$
+ * @since 1.0M8
+ */
+@Unstable
+public interface Gene extends VocabularyProperty
+{
+    /**
+     * Return gene Ensembl ID.
+     *
+     * @return id
+     */
+    @Override
+    String getId();
+
+    /**
+     * Return gene symbol.
+     *
+     * @return symbol
+     */
+    @Override
+    String getName();
```

Comment:
we may refer to those as "names" in UI, but in the code `name` is a very generic overused name (duh). I see you rename some methods which had a `symbol` in their name so at least now it is more consistent, but IMHO it woudl be easier to maintian/understand if we use `symbol` instead everywhere